README.md

```diff
@@ -1,32 +1,16 @@
-# Inferno
+![Inferno Logo](https://github.com/siteadmin/inferno/blob/master/public/images/inferno_logo.png "Inferno Logo")
```

Comment:
This image is broken

Reply:
Broken because it doesn't exist on master yet.